Code/GraphMol/QueryAtom.h

```diff
@@ -47,13 +47,21 @@ class RDKIT_GRAPHMOL_EXPORT QueryAtom : public Atom {
     }
   };
   QueryAtom(const QueryAtom &other) : Atom(other) {
-    dp_query = other.dp_query->copy();
+    if (other.dp_query) {
```

Comment:
This looks like a bug fix or was it introduced by the SetQuery(nullptr)?

Reply:
bug fix